fixtures/data.yaml

```diff
@@ -0,0 +1,1073 @@
+users:
+  - user_id: "john_doe"
```

Comment:
just to be sure, these users are fake so it won't be possible to login with them, right ? or do you have a way to bypass the OFF login ?